manifests/05-operator.yaml

```diff
@@ -22,10 +22,10 @@ spec:
           imagePullPolicy: Always
           resources:
             requests:
-              memory: "100Mi"
-              cpu: "10m"
+              memory: "400Mi"
+              cpu: "30m"
```

Comment:
This is a very small amount of CPU. I'm in favor of upping this to 100m

Reply:
sure, we can do that

Reply:
We should just remove the CPU bits, this will throttle the operator.  But I'm OK w/ this being in for now given we had it before.